fabric_plugin/tests/tests.py

```diff
@@ -221,14 +231,25 @@ def test_task_properties(self):
         instance = self.env.storage.get_node_instances()[0]
         self.assertEqual(instance.runtime_properties['arg'], 'value')
 
-    def test_run_commands(self):
+    def _test_run_commands(self, use_sudo=False):
         commands = ['command1', 'command2']
-        self._execute('test.run_commands', commands=commands)
+        self._execute(
+            'test.run_commands',
+            commands=commands,
+            use_sudo=use_sudo)
```

Comment:
if you are passing the `use_sudo` to the shared test func, you can actually remove the asserts from the two calling functions and simply add a line here
`self.assertIs(use_sudo, self.mock.settings_merged['use_sudo'])`